templates/edit_mod.html

```diff
@@ -51,10 +51,15 @@ <h1>You haven't finished filling out your mod listing!</h1>
     {% endif %}
 
     <div class="container lead">
+        <div class="row">
+            <h1>Edit Mod</h1>
+        </div>
         <div class="row">
             <div class="col-md-8">
-                <h1 title="{{ mod.name }}">Edit {{ mod.name }}</h1>
-                <input type="text" class="form-control input-block-level" name="short-description" value="{{ mod.short_description }}" placeholder="Short description..." />
+                <label for="name" class="text-muted">Name:</label>
+                <input type="text" class="form-control input-lg" name="name" value="{{mod.name}}" />
```

Comment:
What do you think about adding `maxlength` for those fields where we enforce a limit in the backend, so we prevent it right away, before the user hits a 400 screen that doesn't explain what the issue is?
```suggestion
                <input type="text" class="form-control input-lg" name="name" maxlength="100" value="{{mod.name}}" />
```